src/util/treeMiscHelpers.js

```diff
@@ -309,3 +309,20 @@ export const addNodeAttrs = (nodes, newAttrs) => {
     }
   });
 };
+
+/**
+ * Remove attrs from the `nodes` data structure.
+ * @param {Array} nodes
+ * @param {Array} attrsToRemove
+ */
+export const removeNodeAttrs = (nodes, attrsToRemove) => {
+  nodes.forEach((node) => {
+    if(!node.node_attrs) return;
+
+    attrsToRemove.forEach((attrName) => {
+      if(attrName in node.node_attrs){
+        delete node.node_attrs[attrName];
+      }
```

Comment:
You can drop the conditional - `delete` doesn't throw if the `attrName` isn't a property

Reply:
Oops, Python bleeding into JavaScript...dropped conditional in aead1cf13c4000e7d76e090877932018268b9820.